lib/cipher/symmetric/rectangle.ts

```
@@ -0,0 +1,268 @@
+        throw new CipherError('INVALID_KEY_LENGTH', 'RECTANGLE key must be 80 or 128 bits (10 or 16 bytes).')
+    }
+    const inBytes = parseHex(input, 'RECTANGLE input')
+    if (inBytes.length === 0) {
+        return { output: '', outputEncoding: 'hex', steps: [], metadata: METADATA, durationMs: performance.now() - start }
+    }
+
+    const is128 = keyBytes.length === 16
+    const variant = is128 ? 'RECT128' : 'RECT80'
+
+    // PKCS#7 padding for sub-8-byte inputs
+    let padded = [...inBytes]
```

ESLint

[error] 'padded' is never reassigned. Use 'const' instead.

(prefer-const)